### DISCUSSION TITLE
Help with optimizing config

### DISCUSSION THREAD

**Discussion by majian159** [swar/Swar-Chia-Plot-Manager#528] [General]
max_plots: 999
    temporary_directory: /data/cache
    temporary2_directory:
    destination_directory: /data/plots/sdd
    size: 32
    bitfield: true
    threads: 2
    buckets: 128
    memory_buffer: 4000
    max_concurrent: 10
    max_concurrent_with_start_early: 11
    stagger_minutes: 10
    max_for_phase_1: 10
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false

**Comment by Lyushen** [comment]
Wow, you actually bottlenecking everything
stagger_minutes: 60
max_for_phase_1:  4 #(im using max 5 in total of 3 tasks with my 10 cores of 10850k) but still debating between 3 and 5, only if you have similar cpu u can run 4, if anything lower tier or AMD-one, set it to 2)
concurrency_start_early_phase: 4
concurrency_start_early_phase_delay: 10
